code/modules/surgery/organs/organ_stump.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/organ/external/stump
 	name = "limb stump"
 	icon_name = ""
+	var/can_intake_reagents = 0
```

Comment:
<a href=https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/carbon/human/human.dm#L1248-L1253>It's used here, albeit super-indirectly.</a>

This is to prevent a runtime when trying to eat/drink something while you've got a limb stump for a head.